medusa/classes.py

```diff
@@ -15,7 +15,7 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with Medusa. If not, see <http://www.gnu.org/licenses/>.
-
+"""Classes module."""
```

Comment:
Yeah I agree. Problem here is that this is kind of collection of unrelated classes. So the file in it self shouldn't exist imo.